static/js/info_overlay.js

```diff
@@ -149,6 +150,135 @@ This text won't be visible until the user clicks.
     },
 ];
 
+const search_operator_rows = [
+    {
+        operator_key: "stream",
+        operator_value: "stream",
+        effect: "Narrow to messages on stream",
+    },
+    {
+        operator_key: "topic",
+        operator_value: "topic",
+        effect: "Narrow to messages with topic",
+    },
+    {
+        operator_key: "pm-with",
+        operator_value: "email",
+        effect: "Narrow to private messages with",
+    },
+    {
+        operator_key: "group-pm-with",
+        operator_value: "email",
+        effect: "Narrow to group private messages with",
+    },
+    {
+        operator_key: "sender",
+        operator_value: "email",
+        effect: "Narrow to messages sent by",
+    },
+
+    {
+        ignore_operators: true,
+        operator_key: "sender",
+        operator_value: "me",
+        effect: "Narrow to messages sent by you.",
+    },
+    {
+        operator_key: "near",
+        operator_value: "id",
+        effect: "Center the view around message ID",
+    },
+    {
+        operator_key: "id",
+        operator_value: "id",
+        effect: "Narrow to just message ID",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "streams",
+        operator_value: "public",
+        effect: "Search all public streams in the organization.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "alerted",
+        effect: "Narrow to messages with alert words.",
+    },
+    {
+        ignore_operators: true,
+        operator_key: "is",
+        operator_value: "mentioned",
+        effect: "Narrow to messages that mention you.",
+    },
+    {
+        ignore_operators: true,
```

Comment:
`ignore_operators` doesn't sound as a intuitive variable name. Any suggestions for it would definitely help!
Basically, `ignore_operators` key is passed so that the handlebar can render it without coloring the operators: 
Such as: 
![Screenshot_20210609_224150](https://user-images.githubusercontent.com/53977614/121400227-f8a6c380-c974-11eb-8c30-053653a757c1.png)

Reply:
I think we won't need this parameter if you go with the proposal above, but if I'm wrong, I'd call invert the meaning, and instead have a `interpolate_operator: "email"` property for the ones where this sort of interpolation is happening.